extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -160,6 +170,11 @@ class AmpAccordion extends AMP.BaseElement {
 
       header.addEventListener('click', this.clickHandler_.bind(this));
       header.addEventListener('keydown', this.keyDownHandler_.bind(this));
+      section.addEventListener('rendersubtreeactivation', event => {
+        // Event occurs on the content element whose parent is the section to open.
+        const section = dev().assertElement(event.target.parentElement);
+        this.toggle_(section, ActionTrust.HIGH, /* force expand */ true);
```

Comment:
Hmmm. Very interesting. But I'd say since scrolling is low-trust gesture, so should be activation here. That should critically go into a test as well.